lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -522,6 +522,21 @@ sub _stash_clone_info ($self, $job) {
         });
 }
 
+sub gru_tasks_items($self, $job){
```

Comment:
If it's not supposed to be called from anywhere else, name it `_gru_tasks_items`.